libqpdf/QPDFWriter.cc

```diff
@@ -3165,3 +3211,82 @@ QPDFWriter::writeStandard()
         assert(this->md5_pipeline == 0);
     }
 }
+
+void
+QPDFWriter::writePCLm()
+{
+    if (this->deterministic_id)
+    {
+        pushMD5Pipeline();
+    }
+
+    // Start writing
+
+    writePCLmHeader();
+    writeString(this->extra_header_text);
+
+    // Image transform stream content for page strip images.
+    // Each of this new stream has to come after every page image
+    // strip written in the pclm file.
+    std::string image_transform_content = "q /image Do Q\n";
```

Comment:
Is the literal name `/image` meaningful? In other words, does this work in general, or does it depend upon the incoming file looking a certain way?